examples/3_Advanced/coil_force_optimization/coil_forces.py

```diff
@@ -94,9 +100,20 @@
 base_coils = coils[:ncoils]
 bs = BiotSavart(coils)
 bs.set_points(s.gamma().reshape((-1, 3)))
+calculate_on_axis_B(bs, s)
```

Comment:
it's not really the magnetic axis that you're calculating the the modB average right?